source/MaterialXTest/RenderGLSL.cpp

```diff
@@ -162,6 +153,86 @@ void GlslShaderRenderTester::createValidator(std::ostream& log)
     REQUIRE(initialized);
 }
 
+// If these streams don't exist add them for testing purposes
+//
+void addAdditionalTestStreams(mx::MeshPtr mesh)
```

Comment:
For unit tests we ask for some additional streams which we used to create by default in the "sample" obj loader. Now we do the same work here.